draft-ietf-idr-5g-edge-service-metadata-24.xml

```diff
@@ -321,16 +328,23 @@
 
         <section title=" Metadata Path Attribute Processing">
           <t>A BGP speaker that advertises a path received from one of its
-          neighbors SHOULD advertise the BGP Metadata Path attribute received
+          neighbors SHOULD advertise the BGP Metadata Path Attribute received
           with the path without modification as long as the BGP Metadata Path
-          attribute was acceptable. If the path did not come with a BGP
-          Metadata Path attribute, the speaker MAY attach a BGP Metadata
+          Attribute was acceptable. If the path did not come with a BGP
+          Metadata Path Attribute, the speaker MAY attach a BGP Metadata
           Attribute to the path if configured to do so.</t>
+	  <!-- XXX JMH - this is a place where the procedures need to be
+	  subject to the Limited Domain principle.  How do you want to discuss
+	  that check here? -->
```

Comment:
How about changing the wording to the following? 

"A BGP speaker that advertises a path received from one of its neighbors within the same defined domain SHOULD advertise the BGP Metadata Path attribute received with the path without modification as long as the BGP Metadata Path attribute was acceptable. If the path did not come with a BGP Metadata Path attribute, the speaker MAY attach a BGP Metadata Attribute to the path if configured to do so, provided that the modification adheres to the domain's policies and security guidelines."

Reply:
works for me. In next push.